luaui/Widgets/cmd_commandinsert.lua

```diff
@@ -19,7 +19,7 @@ function widget:GameStart()
   widget:PlayerChanged()
 end
 
-function widget:PlayerChanged(playerID)
+function widget:PlayerChanged()
```

Comment:
unused call-in args are there partially so that people copy-pasting the call-in (which is how most people start out) know what's available